CHANGELOG.md

```diff
@@ -15,6 +57,7 @@ See the [release notes](https://puppet.com/docs/pdk/latest/release_notes.html) f
 **Merged pull requests:**
 
 - \(maint\) Default PDK::TEMPLATE\_REF to PDK::VERSION [\#405](https://github.com/puppetlabs/pdk/pull/405) ([rodjek](https://github.com/rodjek))
+- 1.3.2 Release Prep [\#404](https://github.com/puppetlabs/pdk/pull/404) ([HelenCampbell](https://github.com/HelenCampbell))
```

Comment:
This seems silly but (shrug)

Reply:
Why? It was merged for the `v1.3.2` release. 🤷‍♂️

Reply:
In the context of a CHANGELOG communicating to humans what they need to be aware of with a new release it seems like noise, but I understand the limitations of automated CHANGELOG generation.